.changeset/hot-stingrays-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sumup/icons": minor
```

Comment:
I think treating this as a patch is better since it fixes the visual balance without adding any new functionality.

```suggestion
"@sumup/icons": patch
```